WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabViewModel.cs

```diff
@@ -124,8 +124,6 @@ public CoinJoinTabViewModel(WalletViewModel walletViewModel)
 
 		public override void OnOpen()
```

Comment:
@danwalmsley This function is not called after I merged the PR with the submodule removed. I can fix it with a hack but this is indicating that the logic is broken with the `WasabiDocumentTabViewModel`. 

For HistoryTab it is called because I am using the `WalletActionViewModel.DisplayActionTab()`. As you can see here:
https://github.com/molnard/WalletWasabi/blob/b5f6eef84ad247c3949f412362d576c005b59c16/WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs#L85

We should use some comprehensible logic here. When should Open called? I think after the document is added. When should Select called? After Tab is clicked.